sr_example/scripts/sr_example/hand_and_arm_examples/sr_right_hand_arm_ef_pos.py

```diff
@@ -82,62 +74,30 @@
 #         @param end_effector_link - name of the end effector link.
 #         @param alternative_method - use set_joint_value_target instead of set_pose_target.
 #         @param custom_start_state - specify a start state different than the current state.
-pose_1 = [1.1, 0.5, 1.2, -0.9, 0.0, -0.0]
+raw_input("Press Enter to continue...")
+pose_1 = [0.9, 0.16, 0.95, -0.99, 8.27, -0.0, 1.4]
```

Comment:
if we're using "goal" naming in the previous one, please use it here too. Also please say its for the arm.

Reply:
maybe something like "example_goal_0"?

Reply:
same for goal below